app/src/main/java/com/powerhungers/fomezero/feature/login/di/LoginModule.kt

```diff
@@ -16,7 +17,8 @@ val loginModule = module {
         LoginRepositoryImpl(
             remoteDataSource = LoginRemoteDataSource(
                 service = get<Retrofit>().create(LoginService::class.java)
-            )
+            ),
+            localDataSource = LoginLocalDataSourceImpl(get())
```

Comment:
```suggestion
            localDataSource = LoginLocalDataSourceImpl(sharedPreferenceHelper = get())
```
é uma boa pratica especificar o que vc esta esperando receber no construtor da classe

Reply:
outro ponto dessa classe é que a injeção poderia ser feito toda dentro do module de viewModel, da mesma forma que vc fez na injeçao de dependencia da feature de registration